libmamba/CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ if(ENABLE_TSAN)
     add_link_options(-fno-omit-frame-pointer -fsanitize=thread)
 endif()
 
+option(ENABLE_USAN "Enable Undefined-Sanitizer (currently only supported on GCC and Clang)" OFF)
```

Comment:
This is [Undefined Behavior Sanitizer (ubsan)](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html), and I think this could be changed in this PR as well.

Reply:
I applied consistent naming, and changed other options as well to what is usually used.